src/handlers/Chrome70.ts

```diff
@@ -677,7 +677,8 @@ export class Chrome70 extends HandlerInterface
 
 			logger.debug('receive() [trackId:%s, kind:%s]', trackId, kind);
 
-			const localId = rtpParameters.mid || String(this._mapMidTransceiver.size);
+			const nextMSid = this._remoteSdp!.getNextMediaSectionIdx();
+			const localId = rtpParameters.mid || nextMSid.reuseMid || String(nextMSid.idx);
```

Comment:
I fail to understand why we are using (or maybe using) reuseMid here.

Reply:
in #195 , because getNextMediaSectionIdx() looks like it will return a closed mediaSection, so reuse it